src/main/java/com/buyanywhere/productcatalog/models/Category.java

```diff
@@ -1,30 +1,32 @@
 package com.buyanywhere.productcatalog.models;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Category {
-	private long id;
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
The default strategy is GenerationType.AUTO, therefore if you just add `@GeneratedValue` without parenthesis nor parameters this will work

Reply:
Parameters for the annotation removed.